src/main/pages/che-6/user-guide/editor_code_assistance.adoc

```diff
@@ -10,57 +10,82 @@ folder: che-6/user-guide
 
-You can split the consoles into multiple panes. This allows easier navigation when trying to see different console outputs at the same time. *Split Vertical* and *Split Horizontal* can be selected through the drop down menu accessible by at the top right of the console area. To put new console in a newly created pane select the open area below the tabs area.
+* To split the editor into multiple panes, right-click in the editor and select either *Split Vertical* or *Split Horizontal* on the drop down menu.
 
-*Fullscreen Mode for Panes*
+[id="splitting-the-consoles-into-multiple-panes"]
+=== Splitting the consoles into multiple panes
 
-Editor tabs and console outputs can be displayed fullscreen.
+You can split the consoles into multiple panes. This allows easier navigation when trying to see different console outputs at the same time. 
 
-*Fullscreen for Editors*
+* To split the consoles into multiple panes, select *Split Vertical* or *Split Horizontal* from the drop down at the top right of the console area. 
 
-Editor tabs can easily get displayed in fullscreen mode by double-clicking on one of the editor’s tabs. All other panes will be collapsed when you do this. To exit the fullscreen mode, double-click again on the tab.
+* To put the new console in a newly created pane, select the open area below the tabs.
 
-*Fullscreen for Consoles*
+[id="fullsreen-mode-for-panes"]
+=== Fullscreen mode for panes
 
-Console outputs and processes tabs can be displayed in fullscreen mode by double-clicking on one of the terminal’s (or output’s) tab to maximize it. All other panes will be collapsed when you do this. To exit the fullscreen mode, double-click again on the tab.
+The editor tabs and console outputs can be displayed fullscreen.
 
-You can also use the quick pane option displayed in the top right corner of the pane to move to fullscreen mode.
+[id="displaying-the-editor-in-fullscreen"]
+=== Displaying the editor in fullscreen 
 
-*Preview HTML Files*
+* To display the editor in fullscreen mode, double-click on one of the editors tabs. All other panes will be collapsed when you do this. 
 
-You can preview an HTML file by right-clicking it in the project explorer and selecting preview from the popup menu.
+* To exit fullscreen mode, double-click again on the tab.
 
-*Preview Images*
+[id="fullscreen-for-consoles"]
+=== Fullscreen for consoles
 
-You can easily preview image by simple double-clicking it in the project explorer. The image will be opened in the editor area. Also you can open the image in the dedicated browser tab by right-clicking the image in the project explorer and selecting *Preview* from the popup menu.
+* To display the console outputs and processes tabs in fullscreen mode, double-click on one of the terminals or outputs tab to maximize it. All other panes will be collapsed when you do this. 
```

Comment:
will be collapsed/collapse

Reply:
Suggest introductory line at the beginning and then the 2 bullet points.
To display the console outputs and processes tabs in fullscreen mode:
* Double...
* Select... 
The last point could stay as a sentence, without bullets.